modules/ROOT/pages/advanced-templates.adoc

```diff
@@ -0,0 +1,120 @@
+= Advanced Template plugin
+:navtitle: Advanced Template
+:description: Create and edit complex templates in {productname} for use with {productname}.
+:description_short: Create and edit templates.
+:keywords: plugin, advanced template, templates, advtemplate.
+:pluginname: Advanced Template
+:plugincode: advtemplate
+:plugincategory: premium
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.4v.adoc[]
+
+The Advanced Template Premium plugin enables the creation and editing of complex, interactive templates using {productname}.
```

Comment:
Unfortunately, we do not currently support template **editing**. Once a template has been created, it can only be previewed. To update the content of the template, users will need to insert the template into the editor content, make edits, select the appropriate region, and then create a new template.

Reply:
To quote my commit message:

> Edited out references to editing templates.
> 
> Added IMPORTANT admonition outlining using Save as Template… command as editing template workaround.